implementation/scipamato/build.gradle.kts

```
@@ -0,0 +1,117 @@
+import org.jetbrains.kotlin.gradle.plugin.KotlinPlatformJvmPlugin
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+import org.springframework.boot.gradle.tasks.bundling.BootJar
+import org.unbrokendome.gradle.plugins.testsets.TestSetsPlugin
+
+
+plugins {
+    Lib.springBootPlugin().run { id(id) version version } apply false
+    Lib.springDependencyManagementPlugin().run { id(id) version version }
+    Lib.kotlinJvmPlugin().run { kotlin(id) version version }
+    Lib.kotlinSpringPlugin().run { kotlin(id) version version } apply false
+    Lib.lombokPlugin().run { id(id) version version }
+    idea
+    jacoco
+    Lib.testSetsPlugin().run { id(id) version version }
+}
+
+java {
+    version = JavaVersion.VERSION_11
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
+
+extra["spring.cloudVersion"] = "Greenwhich.SR1"
+
+dependencyManagement {
+    imports {
+        mavenBom(SpringBootPlugin.BOM_COORDINATES)
+    }
+}
+
+allprojects {
+    group = "ch.difty"
+    version = "1.1.7-SNAPSHOT"
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
```

```suggestion
        mavenCentral()
        maven { url = uri("http://jaspersoft.jfrog.io/jaspersoft/third-party-ce-artifacts") }
```

The repository above is necessary to prevent the following build error:
```
> Could not resolve all files for configuration ':core-web:compileClasspath'.
   > Could not find com.lowagie:itext:2.1.7.js6.
```

you probably did not encounter the problem, because the `itext` dependency was already installed in your local maven repo ;-)

---

Thanks @jcornaz . You're educated guess was spot on.

In addition I just found that `gradlew :core-web:bootRun` currently fails (while the same for `public-web:bootRun` works). I'm currently investigating and will push asap.